### DISCUSSION TITLE
Listing the current installed version

### DISCUSSION THREAD

**Discussion by winston-programmable** [netbox-community/netbox#6483] [Q&A] [answered]
Pretty new to this so forgive the noddy question. What command can be used to display the current running versions. Online searches have not yielded any results

**Comment by drmsoffall** [comment, marked as answer]
Can you be more specific?  Versions of what?  From what environment?

**Reply by winston-programmable** [reply]
Sure, i have inherited a server running Ubuntu 18.04 and has an instance of Netbox. I would like determine what version of Netbox is running. E.g. the latest version available here on git is 2.11.4 so, i am trying to find out what 2.x.x is running on my host, similar to executing "python3 -V" to determine what version of python. Hope that is clear

**Reply by drmsoffall** [reply]
The easiest way is to go to the web interface and check the footer.

The "most accurate" on-system way is probably to use Python from within the NetBox virtual environment:
```
$ . venv/bin/activate
(venv) $ python netbox/manage.py nbshell
>>> from netbox import settings
>>> settings.VERSION
'2.11.2'
```

The hacky way if you need it for scripting or something is:
```
grep '^VERSION' netbox/netbox/settings.py | cut -d \' -f 2
2.11.2
```

**Reply by winston-programmable** [reply]
Perfect. Thank U!

**Reply by ryanmerolle** [reply]
its also in the webui at the bottom right hand corner, or via the api at `<url>/api/status/`

Example: https://demo.netbox.dev/api/status/
